swig/sundials/fsundials_stepper.i

```diff
@@ -0,0 +1,26 @@
+// ---------------------------------------------------------------
+// Programmer: Steven B. Roberts @ LLNL
+// ---------------------------------------------------------------
+// SUNDIALS Copyright Start
+// Copyright (c) 2002-2024, Lawrence Livermore National Security
+// and Southern Methodist University.
+// All rights reserved.
+//
+// See the top-level LICENSE and NOTICE files for details.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+// SUNDIALS Copyright End
+// ---------------------------------------------------------------
+// Swig interface file
+// ---------------------------------------------------------------
+
+%{
+#include "sundials/sundials_stepper.h"
+%}
+
+// TODO(SBR): Can someone confirm this is correct
+%apply void* { SUNStepper };
```

Comment:
This is correct unless we want users to be able to make a custom SUNStepper in Fortran.

Reply:
Additional note... since we do not make the `SUNStepper` struct public, users cannot fully customize a `SUNStepper` anyways (only the ops), so this makes sense.